language/move-model/src/lib.rs

```diff
@@ -265,6 +270,115 @@ fn add_move_lang_errors(env: &mut GlobalEnv, errors: Errors) {
     }
 }
 
+fn script_into_module(compiled_script: CompiledScript) -> CompiledModule {
```

Comment:
I wonder if you should mark this as deprecated or at least add a comment here explaining that this should be removed in the future.